From c84e90c4209258941a1f351f3de84fb9d9b7331c Mon Sep 17 00:00:00 2001
From: Qiang Zhu <alecfans@gmail.com>
Date: Fri, 7 Jul 2023 17:51:28 -0700
Subject: [PATCH] check if pyshtools is compatible #222

---
 requirements.txt | 2 +-
 setup.py         | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/requirements.txt b/requirements.txt
index b8e2df1..40634c0 100644
--- a/requirements.txt
+++ b/requirements.txt
@@ -7,4 +7,4 @@ scipy>=1.7.3
 importlib_metadata>=1.4
 ase>=3.18.0
 numba>=0.55.2
-pyshtools==4.10
+pyshtools>=4.10.3
diff --git a/setup.py b/setup.py
index 9d98c3e..bf4f089 100755
--- a/setup.py
+++ b/setup.py
@@ -52,7 +52,7 @@
         'numba>=0.55.2', #now supports numpy 1.22
         'scipy>=1.7.3',
         'importlib_metadata>=1.4',
-        'pyshtools==4.10',
+        'pyshtools>=4.10.3',
         #"openbabel>=3.0.0",
     ],
     python_requires=">=3.7, <=3.11", #add the restriction for now issue #189
